### PR TITLE
Add French homepage template (#7282)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Add content card based French homepage.

## Issue / Bugzilla link

See #7282.

## Testing
https://bedrock-demo-6.oregon-b.moz.works/fr/

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@alexgibson This isn't consuming the new cards in www-admin and I don't know why. Can you have a look please?

**Comment by alexgibson** [comment]
> @alexgibson This isn't consuming the new cards in www-admin and I don't know why. Can you have a look please?

I did some debugging, and it looks like when `Dev=True`, bedrock is looking for cards in the [`dev-processed`](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L1295) as opposed to `master-processed`. I guess this is since we changed the branch names so you could deploy separetely to demo. You might need to manually update the `dev` branch against `master` as changes are merged.

**Comment by alexgibson** [comment]
Passing note: you may need to update functional tests: https://github.com/mozilla/bedrock/blob/master/tests/functional/test_home.py

**Comment by stephaniehobson** [comment]
Ah yeah, that fixed it, thanks.

**Comment by stephaniehobson** [comment]
Okay, this is ready for review!

File: bedrock/mozorg/templates/mozorg/home/home-fr.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        include_highres_image=True
+      )}}
+
+      <div class="mzp-l-card-half">
+        {{ content_card('card_6') }}
+        {{ content_card('card_7') }}
+      </div>
+
+      {{ billboard(
+        title=_('Technologies web open source :'),
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-fr.html]: It's French. In English we'd say "Open-source web technologies:"

French... it's like English but backwards?

**Comment by stephaniehobson** [comment]
Moved the test into this PR.

**Pull request merged**